ITR/interfaces.py

```diff
@@ -459,12 +598,12 @@ class ICompanyData(PintModel):
     country: Optional[str]
 
     emissions_metric: Optional[EmissionsMetric]    # Typically use t CO2 for MWh/GJ and Mt CO2 for TWh/PJ
-    production_metric: Optional[ProductionMetric]  # Optional because it can be inferred from sector and region
+    production_metric: Optional[production_quantity(_production_units)]  # Optional because it can be inferred from sector and region
```

Comment:
what makes `production_quantity` more appropriate here?

Reply:
Something is screwed up with the GitHub state.  ProductionMetric is correct.

Reply:
agree, again, i commented on earlier commit, but you modified in a later commit.
no problem here